ios/Video/RCTVideo.m

```diff
@@ -883,7 +883,9 @@ - (void)setPictureInPicture:(BOOL)pictureInPicture
   if (_pictureInPicture == pictureInPicture) {
     return;
   }
-  
+
+  [self setupPipController];
```

Comment:
I believe you should call this after you set `_pictureInPicture` to `pictureInPicture`. It should look like this:

```
[self setupPipController];
_pictureInPicture = pictureInPicture;
```